src/Features/Core/Portable/AddConstructorParametersFromMembers/AddConstructorParametersFromMembersCodeRefactoringProvider.AddConstructorParametersCodeAction.cs

```diff
@@ -101,6 +101,16 @@ public override string Title
                     }
                 }
             }
+
+            /// <summary>
+            /// A metadata name used by telemetry to distinguish between the different kinds of this code action.
+            /// This code action will perform 2 different actions depending on if missing parameters can be optional.
+            /// 
+            /// In this case we don't want to use the title as it depends on the class name for the ctor.
+            /// </summary>
+            internal string ActionName => _missingParameters[0].IsOptional
```

Comment:
I think once we decide to add more of these refactorings, we need to make this telemetry distinguishing variable a required part of the action.  Since we can't use the title in cases like this